debian/control

```diff
@@ -32,10 +32,14 @@ Description: Provisioning backend for Homestead, the Cassandra powered HSS gatew
 
 Package: homer-cassandra
 Architecture: any
-Depends: clearwater-cassandra, clearwater-infrastructure
+Depends: clearwater-cassandra, clearwater-infrastructure, crest-prov
```

Comment:
Should this actually be a strict dependency, as opposed to a recommends? I don't think homer/homestead-prov-cassandra actually requires crest-prov to function?